.github/workflows/e2e.yml

```diff
@@ -22,4 +22,4 @@ jobs:
         if: always()
```

Comment:
Is this necessary?

Reply:
According to this article, I assume it is:
https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions